### DISCUSSION TITLE
[Feature] List similar pictures of the one you are currently looking at

### DISCUSSION THREAD

**Discussion by chriexpe** [immich-app/immich#7809] [Feature Request]
### The feature

Let's suppose you want to find a picture of your dog, you search "dog" but CLIP throws a ton of different results, so you have two options, be more specific or, this feature suggestion, where you open one image of your dog and by swiping/scrolling down (or going to _**Info**_ tab) it would show pictures that looks similar to the one you're looking at, kinda like when you search a image on Google, you open one that you liked and below it there will be more similar to that.
In addition to this, maybe in a near future a google lens style of search where you select on the picture the part you want CLIP to search similar object (or like what Yandex does).

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4509
